Tasks/VsTest/runvstest.ts

```diff
@@ -21,6 +21,10 @@ if (osPlat !== 'win32') {
 
     try {
         utils.Helper.setConsoleCodePage();
+        const blockRun = isMultiConfigOnDemandRun();
+        if (blockRun) {
+            tl.setResult(tl.TaskResult.Failed, tl.loc('OnlyWindowsOsSupported'));
```

Comment:
Change the message

Reply:
to this MultiConfigNotSupportedWithOnDemand

Reply:
Done. Thanks